docs/topics/logging.txt

```diff
@@ -261,12 +265,37 @@ First, here's a simple configuration which writes all request logging from the
 If you use this example, be sure to change the ``'filename'`` path to a
 location that's writable by the user that's running the Django application.
 
-Second, here's an example of a fairly complex logging setup, configured using
+Second, here's an example of how to make the logging system print all of
+Django's logging to the console based on the value of an environment variable.
+It overrides the fact that ``django.request`` and ``django.security`` don't
+propagate their log entries. It may be useful during local development. Set the
+environment variable ``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging
```

Comment:
Ok - if we keep it in the example, I think it requires a bit more explanation. Something like "by default, this config only sends messages of level `INFO` or higher to the console, and Django itself does not currently log any such messages. Set the environment variable `DJANGO_LOG_LEVEL=DEBUG` to see all of Django's debug logging."